### DISCUSSION TITLE
DNS queries

### DISCUSSION THREAD

**Discussion by fidoman** [wiresock/proxifyre#43] [Q&A]
Hello, does proxifyre catch and route to proxy DNS queries from apps? Cannot get working access to proxied servers that does not have public DNS name (but that can be resolved by proxy, as browsers do).

**Comment by wiresock** [comment]
DNS queries in Windows are handled by the DNSCACHE service. You can try adding this service to the list of proxied applications. However, please note that since DNSCACHE is a service, Proxifyre should either run as a service itself or with administrator privileges to properly intercept and route the DNS queries.

**Reply by fidoman** [reply]
I had added "svchost.exe" to apps but no effect.
(also restarted service by killing corresponding process)
also enabled UDP
         "supportedProtocols": ["TCP", "UDP"]

**Reply by wiresock** [reply]
I actually suggested adding DNSCACHE, not svchost.

**Reply by fidoman** [reply]
added "DNSCACHE", no effect

**Reply by wiresock** [reply]
How did you start ProxiFyre— as a regular user, administrator, or service? Also, is there a chance that you have an IPv6 DNS server configured?

**Reply by fidoman** [reply]
Probably my proxy do not pass UDP as I see requests from nslookup.exe in proxifyre log but there is no packets on server side.
